content/docs/setup/kubernetes/additional-setup/requirements/index.md

```diff
@@ -34,6 +34,10 @@ cluster must satisfy the following requirements:
     plain TCP traffic unless the port [explicitly](https://kubernetes.io/docs/concepts/services-networking/service/#defining-a-service)
     uses `Protocol: UDP` to signify a UDP port.
 
+    {{< warning >}}
```

Comment:
I'm not sure this warning is appropriate here because this section is talking about port names in a Kubernetes `Service` while your warning is about port names in a `Gateway` config.

Isn't this sufifcient? https://istio.io/docs/ops/traffic-management/deploy-guidelines/#configuring-multiple-tls-hosts-in-a-gateway

Reply:
Thanks. I looked but didn't see this section before. 

Is there a way to highlight this better? I know of at least two separate reported issues where users have encountered issues while using the same port names for HTTPS. One on Istio Github, which is linked, and one on the maistra Jira, so I wonder if we can highlight this a bit more.